### PR TITLE
python310Packages.pydevd: fix darwin

### PR DESCRIPTION
**Pull request by happysalada**
###### Description of changes

@onny I've got 2 tests reliably failing on darwin. Are you ok to disable them ? Or do you want me to put this behind a darwin flag ?

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by happysalada** [comment]
I've cured my laziness and disabled the tests only for darwin. Let me know if anything.

**Comment by happysalada** [comment]
@onny let me know about this when you have a moment.

File: pkgs/development/python-modules/pydevd/default.nix [hunk 1]
```diff
@@ -48,6 +49,9 @@ buildPythonPackage rec {
     # AssertionError pydevd_tracing.set_trace_to_threads(tracing_func) == 0
     "test_tracing_other_threads"
     "test_tracing_basic"
+  ] ++ lib.optionals stdenv.isDarwin [
```
Comment (onny) [inline pkgs/development/python-modules/pydevd/default.nix:52]: Thank you for testing on darwin and sorry for my late reply. When it's possible you can add a comment on why it is failing or maybe link to an upstream issue

Comment (happysalada) [reply pkgs/development/python-modules/pydevd/default.nix:52]: if you let me know what would be a good summary in your opinion, I'm happy to add a comment

Comment (onny) [reply pkgs/development/python-modules/pydevd/default.nix:52]: Yeah this looks complicated and unfortunately I'm not into pydevd

**Comment by onny** [comment]
Thank you for improving the package on Dariwn :)

**Comment by happysalada** [comment]
Thank you for maintaining the package !

**Pull request merged**